src/button/validate.js

```diff
@@ -127,6 +128,31 @@ export function validateButtonStyle(style : Object = {}) {
             throw new Error(`style.${ BUTTON_STYLE_OPTIONS.TAGLINE } is not allowed for ${ BUTTON_LAYOUT.VERTICAL } layout - got ${ tagline }`);
         }
     }
+
+    validateRegionSpecificButton(style, props.locale);
+
+}
+
+export function validateRegionSpecificButton(style : Object = {}, locale : string) {
+
+    let country = locale.split('_')[1];
+
+    let isInstallmentCountry = Object.keys(INSTALLMENT_COUNTRY).find(elem => {
```

Comment:
Probably if this was an array we could just do `(INSTALLMENT_COUNTRY.indexOf(country) !== -1)`

Reply:
ok.